azure-pipelines.yml

```diff
@@ -8,32 +8,25 @@ trigger:
     - '*'
 
 jobs:
-#- job: CredScan
-#  displayName: "Credential Scan"
-#
-#  pool:
-#    vmImage: "windows-2019"
-#  steps:
-#  - task: CredScan@2
-#    inputs:
-#      toolMajorVersion: 'V2'
-#
-#  - task: PostAnalysis@1
-#    inputs:
-#      AllTools: false
-#      APIScan: false
-#      BinSkim: false
-#      CodesignValidation: false
-#      CredScan: true
-#      FortifySCA: false
-#      FxCop: false
-#      ModernCop: false
-#      PoliCheck: false
-#      RoslynAnalyzers: false
-#      SDLNativeRules: false
-#      Semmle: false
-#      TSLint: false
-#      ToolLogsNotFoundAction: 'Standard'
+- job: CredScan
```

Comment:
Is it possible that CLI and extensions repo share the same yaml section instead of update twice?